bundle/language_sw.properties

```diff
@@ -352,6 +352,8 @@ angal.patient.nextkin                                                       = Mr
 angal.patient.note                                                          = Kidokezo
 angal.patient.parenttogether                                                = Wazazi pamoja
 angal.patient.patientphoto                                                  = Picha ya Mgonjwa
+angal.patient.patientphoto.error.title                                      = Patientfotofel
+angal.patient.patientphoto.error.filenotfound                               = Patientfoto hittades inte.
```

Comment:
There's no need to provide translations as they will come from Transifex once merged on develop, thanks